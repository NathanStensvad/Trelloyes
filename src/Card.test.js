import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Card id="1" title="Card title" content="Stuff" onDeleteCard={() => {}}/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('matches the snapshot', () => {
  const jsonTree = renderer.create(
    <Card id="1" title="Card title" content="Stuff" onDeleteCard={() => {}}/>
  ).toJSON();

  expect(jsonTree).toMatchSnapshot();
});