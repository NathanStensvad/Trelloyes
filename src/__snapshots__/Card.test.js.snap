// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="Card"
>
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3>
    Card title
  </h3>
  <p>
    Stuff
  </p>
</div>
`;
